src/pages/styles.less

```diff
@@ -140,12 +138,40 @@
     @apply shadow-none;
   }
 
-  .transparent-navbar, .transition-navbar:hover {
+  .transparent-navbar,
```

Comment:
Let's revert this line change

Reply:
Nothing really changed there...just text formatting.